### ISSUE TITLE
Shortcuts partly broken: Depend on the order you press the keys

### ISSUE THREAD

**Issue by lkiesow** [opencast/opencast-editor#739]
The shortcuts in the editor depend on the order the keys are pressed on the keyboard, which makes them very unreliable and annoying to use. For example, in the cutting view, `Ctrl + Alt + d` does not work if you press these keys in order, but `Alt + Ctrl + d` does work.

Reproducible in Chrome, Chromium and Firefox on Fedora 35 Cinnamon.

**Comment by Arnei** [comment]
Can't reproduce this. Does not matter if I pres `Ctrl` or `Alt` first.

The one thing i found that does not work in Chrome is pressing `Alt + d` before `Ctrl`, because that moves focus to the adress bar.

Works for me in Chrome and Firefox on Fedora 34.

**Comment by cgruber-uos** [comment]
Same result as Arne reports for Win11: Can not reproduce any difference in `Ctrl + Alt + d` vs. `Alt + Ctrl + d` with FF / Edge / Chrome all 106.x. 

Just pressing `Ctrl + d` first leeds to "edit bookmark" dialog and `Alt + d` leads to browser's file menu.

**Comment by Arnei** [comment]
Closing this because given Clemens and my testing, the issue does not seem to be general, but local to Lars.
